src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/AbstractSyntaxFormattingService.cs

```diff
@@ -16,11 +16,11 @@
 
 namespace Microsoft.CodeAnalysis.Formatting
 {
-    internal abstract class AbstractSyntaxFormattingService : ISyntaxFormattingService
+    internal abstract class AbstractSyntaxFormatting : ISyntaxFormatting
```

Comment:
Nit: The file should be renamed to `AbstractSyntaxFormatting.cs`.

Reply:
yes.  that makes sense.  will do in followup!

Reply:
DOne in https://github.com/dotnet/roslyn/pull/60462